gravitee-apim-console-webui/src/management/api/general/plans/edit/api-general-plan-edit.component.html

```diff
@@ -17,7 +17,7 @@
 -->
 <div>
   <span class="md-headline">
-    <gio-go-back-button [ajsGo]="{ to: portalPlansRoute }"></gio-go-back-button>
```

Comment:
ca we remove component portalPlansRoute var ?

Reply:
The issue is because the variable `portalPlansRoute` doesn't exist anymore in the component.

Reply:
aaa haha ok